### ISSUE TITLE
Is it possible to use helper function via 'from pytest.helper import ...'?

### ISSUE THREAD

**Issue by kepbod** [saltstack/pytest-helpers-namespace#5]
Is it possible to use helper function via `from pytest.helper import ...`?

**Comment by s0undt3ch** [comment]
No. And if we're talking about test modules, you don't need to. Just import `pytest` and use `pytest.helpers.<whatever-helper-function>()`.

If we're talking about a module outside of tests, then this plugin is not applicable.

**Comment by s0undt3ch** [comment]
Please reopen if need be.

**Comment by diazona** [comment]
FWIW I think it would be nice to have this feature. When I have dozens of tests in the same file that all use a helper function, it's a bit annoying to have to type `pytest.helpers.` every time. And while I could do this:

```python
import pytest

whatever_helper_function = pytest.helpers.whatever_helper_function
```

that feels like I'm cheating my way around the import system :stuck_out_tongue: which I know is not really the case, it just looks weird because ordinarily I would _absolutely_ replace that with a `from ... import ...` statement.

That being said, the lack of `from ... import ...` support is certainly not blocking anything, and it's not something I consider a big enough deal to dig into the package and implement it myself (at least, not for now). Just wanted to cast a "vote" to say it's a feature that would be found useful, if someone ever does want to implement it.

**Comment by s0undt3ch** [comment]
The problem is, if we were to implement that, we would have to mess with the import machinery, create a module at runtime, multiple if we namespace the helpers.

Right now, we already abuse by adding the `helpers` attribute to pytest, the import changes, well, that's a lot of magic.

Don't get me wrong, I see the "beauty" in it, but, it would probably complicate things quite a bit...

**Comment by s0undt3ch** [comment]
Reopening the issue for now.

**Comment by diazona** [comment]
Fair enough. I'm familiar with how complicated things can get when hacking the import machinery! If it turns out this feature is just too much trouble to be worth it, I can definitely live with that.
